app/tut/ga/usaideach.md

```diff
@@ -0,0 +1,339 @@
+`2`{.setanta}; Tá `10 / 3`{.setanta} cothrom le `3.333`{.setanta} ach tá `10 // 3`{.setanta} cothrom
+le `3`{.setanta}.
+
+## Gníomhartha Matamaitice
+
+Chonaiceamar roinnt gníomhartha matamaitice cheanna sa teagasc seo, go háirithe
+`rand@mata`{.setanta} agus `slánuimh_rand@mata`{.setanta}, áfach, tá a lán níos mó gníomhartha agus
+luachanna matamaitice ar fáil i *Setanta*.
+
+Seo liosta gearr do na gníomhartha agus luachanna atá ar fáil, bá chóir duit `@mata`{.setanta} a
+úsáid chun iad a roghnú, mar shampla `fréamh@mata`{.setanta}.
+
+### Luachanna
+
+| Ainm | Cur síos | Sampla
+|:------|:-----:|----:|
+| `pí` | An [[tairiseach|constant]] Pí (3.1415...) | `2 * pí@mata * r`{.setanta} |
+| `e` | An tairiseach e (an uimhir Euler) (2.71828...) | `cmhcht@mata(e@mata, 2)`{.setanta} |
+
+### Gníomhartha
+
+| Ainm | Cur Síos agus Sampla |
+|:------|:-----:|
+| `fréamh` | An [[fréamh cearnach|square root]] (√) |
+|          | **m.sh.** `fréamh@mata(4) == 2`{.setanta} |
+| `cearn` | [[Feidhm|function]] cearnach (`x²`) |
+|         | **m.sh.** `cearn@mata(2) == 4`{.setanta} |
+| `dearbh` | Feidhm [[dearbhluacha|absolute value]] (`|x|`) |
+|          | **m.sh.** `dearbh@mata(-2) == 2`{.setanta} |
+| `eas` | Feidhm [[easpónantúile|exponential]] (`e^x`) |
+|       | **m.sh.** `eas@mata(1) == e@mata`{.setanta} |
+| `cmhcht` | Feidhm [[cumhachta|power]] (`x^y`) |
+|          | **m.sh.** `cmhcht@mata(2, 4) == 16`{.setanta} |
+| `log` | Feidhm [[logartaim|logarithm]] |
+|       | **m.sh.** `log@mata(2)`{.setanta} |
+| `logb` | [[Logartam i mbun éigin|logarithm in some base]] |
+|        | **m.sh.** `logb@mata(16, 2) == 4`{.setanta} |
+| `sin`  | Feidhm Sine |
+|        | **m.sh.** `sin@mata(pi@mata/2)`{.setanta} |
+| `cos`  | Feidhm Cosine |
+|        | **m.sh.** `cos@mata(0)`{.setanta} |
+| `tan`  | Feidhm [[Tangaint|Tangent]] |
+|        | **m.sh.** `tan@mata(pi@mata)`{.setanta} |
+| `asin` | Feidhm sine [[inbhéarta|inverse]] |
+|        | **m.sh.** `asin@mata(0)`{.setanta} |
+| `acos` | Feidhm cosine inbhéarta |
+|        | **m.sh.** `acos@mata(pi@mata)`{.setanta} |
+| `atan` | Feidhm tangaint inbhéarta |
+|        | **m.sh.** `atan@mata(0)`{.setanta} |
+| `rand` | Uimhir randamach idir 0 agus 1 |
+|        | **m.sh.** `rand@mata()`{.setanta} |
+| `slánuimh_rand` | Slánuimhir randamach i raon éigin |
+|                 | **m.sh.** `slánuimh_rand@mata(5, 10)`{.setanta} |
+
+# Téacs
+
+Tá a fhios againn faoi conas `+` a úsáid chun dhá píosa téacs a chur le chéile, conas
+`fad`{.setanta} a úsáid chun fad an phíosa téacs a fháil agus conas
+[[slonn innéacs|index expression]] a úsáid chun [[carachtair|characters]] an píosa téacs a roghnú.
+Áfach, is féidir linn a lán níos mó a dhéanamh le téacs. Féachfaimid anois ar roinnt gníomhartha
+úsáideach:
+
+## `go_téacs`
+
+Is féidir linn `go_téacs` a úsáid chun aon luach a athrú go
+[[léiriú téacsúil|textual representation]]. Mar shampla: tá `go_téacs([1, 2, 3])`{.setanta} cothrom
```

Comment:
Níl "cothrom le" ceart anseo, measaim.
Is ionann go_téacs([1, 2, 3]) agus "[1, 2, 3]"

Reply:
Athraithe